Outline.md

```diff
@@ -0,0 +1,137 @@
+# Outline - Build Modern APIs in NodeJS
+
+Welcome to the course, in this course we are going to learn how to build modern APIs using NodeJS. Mainly we will focus on REST API as it will be found all over the world. But we will also try to understand other popular competitors of REST APIs like GraphQL, JSON API, gRPC like techniques. End of this course you will learn - 
+
+-   What is API (Application Programming Interface)
+-   What is REST API (Representation State Transfer)
+-   Why do We Need API
+-   Best Alternatives of REST API and Their Use Cases
+-   How to Implement REST API using Node and Express
+-   How to Implement GraphQL API using GraphQL Yoga or Express GraphQL
```

Comment:
I've also seen Apollo GraphQL as a popular framework in this space. 
We should base our GraphQL framework choice based on which one is most searched as that is likely to draw in the most customers.

Reply:
You may please check these links - 
https://dev.to/vinodchauhan7/which-is-best-apollo-server-vs-express-graphql-vs-graphql-yoga-vs-prisma-vs-apollo-server-express-5g42
https://dev.to/yeeiodev/differences-between-express-graphql-apollo-and-graphql-yoga-3m82

Reply:
You may please check these links -
https://dev.to/vinodchauhan7/which-is-best-apollo-server-vs-express-graphql-vs-graphql-yoga-vs-prisma-vs-apollo-server-express-5g42
https://dev.to/yeeiodev/differences-between-express-graphql-apollo-and-graphql-yoga-3m82